tests/unit/output-parser.test.ts

```diff
@@ -0,0 +1,498 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'mocha';
+import { expect } from 'chai';
+import {
+  parseUpdateTrafficResponse,
+  parseDeployResponse,
+  ParseInputs,
+} from '../../src/output-parser';
+
+describe('#output-parser', () => {
+  describe('#parseUpdateTrafficResponse', () => {
+    const cases: {
+      name: string;
+      input: { stdout: string };
```

Comment:
What's the value of having a nested structure here? Are you thinking of supporting `stderr` for tests too? If not, it might be simpler to denest this.